Automattic/wp-calypso
# State: Create media selected items reducer, action creators and selectors

#### Changes proposed in this Pull Request

* State: Add media library selected items action creator for updating selected items
* State: Add media library selected items reducer with tests
* State: Introduce `getMediaLibrarySelectedItems` selector with tests

#### Testing instructions

* Verify all tests pass.

Note: This PR is part of an iteration to remove the media library selected items store, see #40269 for the full effort that removes it completely. Also see #39379 for the complete media reduxification effort.
